### ISSUE TITLE
Authentication issue with SMB share on Linux

### ISSUE THREAD

**Issue by tazrome** [DFIR-ORC/dfir-orc#52]
Hi,

It seems we have an issue to upload the result archive on a Linux SMB share. We are facing with authentication issue. It's seems like ORC add a \ in the begining of username and this cause a bad authentication.

method=filecopy
mode=sync
operation=move
authscheme=[negotiate or basic]

Great thanks for your work

**Comment by tazrome** [comment]
Hello,

Sorry for the delay... below few more détails. Hopes it will help.

Samba configuration is pretty simple as you describes above...

```
Setup orc v10.0.19
```

```
security = user
...
[upload]
path = /opt/upload
valid users = bob
read only = no
```

```
**With Samba 4.7.1**

The authentication works but I need to prefix a "fake" domain before the login user... something like toto\bob as username

Nothing really relevant in log files even with a log level > 5. I see an authentication error with bad password with the user bob and it's ok with a user anydomain\bob

**Samba 4.9.5**

The authentication works normaly (with only "bob" as user)
```

I tried multiple samba configuration but i'm not sure if it 's related to Orc, Samba or my samba configuration... There is no Windows domain involved with Samba in my setup.

Thanks

Regards

**Comment by sc-anssi** [comment]
Hi,
I honestly don't know what made Samba 4.7.1 fail, but since the issue seems to be fixed in Samba 4.9.5 I don't think it's related to DFIR-Orc. Both Samba versions are pretty old and not supported anymore, you should definitely update if possible.
Regards.
